### PR TITLE
Reorganize bank admin section on homepage

### PR DESCRIPTION
**Pull request by devramsean0**
closes #3893
This PR does:
- Adds the Google Workspace Waitlist card in perks
- Makes the admin section a partial (static_pages/admin) to improve the readability of the section
- Reorganizes the sections
- Gets rid of the disused Stickermule Perk card from displaying

image

### CODE REVIEW

File: app/views/static_pages/index.html.erb [hunk 1]
```diff
@@ -142,97 +142,5 @@
...
-        <%= card_to "Emburse cards", emburse_cards_path %>
-        <%= card_to "Emburse card requests", emburse_card_requests_path %>
-        <%= card_to "Emburse transfers", emburse_transfers_path %>
-        <%= card_to "Emburse transactions", emburse_transactions_path %>
-        <%= render partial: "static_pages/card", locals: { task_path: admin_search_path, human_name: "Search by user" } %>
-        <%= render partial: "static_pages/card", locals: { task_path: "https://hack.af/hcb-knowledgebase", human_name: "Knowledgebase" } %>
-      </ul>
-    </details>
-  <% end %>
+  <%= render partial: "static_pages/admin" %>
```
Comment (garyhtou) [inline app/views/static_pages/index.html.erb]: thank you so much for this!

**Review by garyhtou — APPROVED** [review]
hey @devramsean0! sorry for the delay here.

This is looking great! I love that you pulled the admin section isn't its own partial. it definitely makes the code much more manageable.

The ERB Lint test is failing for a very odd reason. I'm looking into this and will get back to you.

Everything looks great from an engineering perspective. Can you please ask someone from Ops to take a look at the staging deployment, make sure it all looks good from an Ops perspective, and then leave a review on this PR?

**Comment by garyhtou** [comment]
Looks like ERB Lint was erroring because of carriage returns (CRLF). Windows systems use CRLF which is represented by `/r/n`, while Mac and UNIX use LF, represented by `/n`. Since the error thrown by ERB Lint was "[begin_pos location 6975 is out of range for document of size 6896"](https://github.com/Shopify/better-html/blob/main/lib/better_html/tokenizer/location.rb#L14C1-L14C1)", I'm assuming that the extra `\r` was not properly accounted for in their tokenization which causes an out of bounds error.

**Comment by devramsean0** [comment]
Agh. Vscode should be set to LF but it's not perfect ig

**Comment by daisybanks** [comment]
Hey @devramsean0! Can you please reorganize the dashboard to have tasks at the top, followed by information, perks, accounting and then other. 

Additionally, could you please move pending ledger and recurring donations to "other." 

We can also remove raw transactions, bank accounts, bank accounts (2), transaction csvs, HCB codes and set up HCB from the dashboard completely. 

Thanks so much!

**Comment by devramsean0** [comment]
@daisybanks thoughts on it now?

**Comment by daisybanks** [comment]
@devramsean0 I don't see the changes but I'm also not good with Github haha - Can you tell me how to see the final product?

**Comment by devramsean0** [comment]
I forgot Gary / Someone with heroku access has to manually trigger a redeploy lol, lemme ask someone

**Comment by devramsean0** [comment]
Approved by daisy
Screenshot_20230818_224323_Slack

**Review by garyhtou — APPROVED** [review]
Great work! go ahead and merge after the tests pass

**Pull request merged**